### PR TITLE
Rework PRR questionaire

### PR DESCRIPTION
**Pull request by wojtek-t**
This is based on what I've seen when doing the pilot phase with 3 KEPs:
- simplify some of those (given we will get the same answer in 90+% of cases)
- clarify those where people didn't fully understood the intention)
- merge some of them (or add TODOs) to think if they make sense

/assign @johnbelamaric 

@liggitt

### CODE REVIEW

File: sig-architecture/production-readiness.md [hunk 1]
```diff
@@ -15,63 +15,121 @@ aspects of the process.
...
+#### Thoubleshooting
+
+* **What are the known failure modes?**
+
+* **How can those be detected via metrics or logs?**
+
+* **What are the mitigations for each of those failure modes?**
+
+* **What are the most useful log messages and what logging levels to they require?**  
+  TODO(wojtek-t): Is that really a question that makes sense at the KEP level?
```
Comment (johnbelamaric) [inline sig-architecture/production-readiness.md]: No, this is more of a playbook question. We could instead request a playbook at alpha -> beta or beta -> GA, and move troubleshooting there, perhaps along with monitoring details. Or, if we want to keep everything in the KEP then we can make the troubleshooting section only required upon beta -> GA.

File: sig-architecture/production-readiness.md [hunk 2]
```diff
@@ -15,63 +15,121 @@ aspects of the process.
...
+  - [ ] Metrics
+	  - Metric name:
+    - [Optional] Aggregation method:
+    - Components exposing the metric:
+  - [ ] Other (treat as last resort)
+    - Details:
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+
+#### Thoubleshooting
```
Comment (johnbelamaric) [inline sig-architecture/production-readiness.md]: typo Th -> Tr

File: sig-architecture/production-readiness.md [hunk 3]
```diff
@@ -15,63 +15,121 @@ aspects of the process.
...
+
+* **Can the feature be disabled once it has been enabled (i.e. can we rollback
+  the enablement)?**  
+  Describe the consequences on existing workloads (e.g. if this is runtime
+  feature, can it break the existing applications?).
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+
+* **Are there any tests for feature enablement/ disablement?**  
+  TODO(wojtek-t): Do we even have any such tests for existing features currently?
```
Comment (johnbelamaric) [inline sig-architecture/production-readiness.md]: We should have conversion tests for versions of resources, at a minimum.

Comment (wojtek-t) [reply sig-architecture/production-readiness.md]: Added clarification

File: sig-architecture/production-readiness.md (hunk 2 above)
Comment (wojtek-t) [reply sig-architecture/production-readiness.md]: done

File: sig-architecture/production-readiness.md (hunk 1 above)
Comment (wojtek-t) [reply sig-architecture/production-readiness.md]: Makes sense - added a comment about that.

**Pull request merged**